browser_patches/firefox/patches/bootstrap.diff

```diff
@@ -3711,6 +3711,12 @@ index 0000000000000000000000000000000000000000..63754937b3464794227fe894b3d6057f
 +
 +const {Helper} = ChromeUtils.import('chrome://juggler/content/Helper.js');
 +const {NetUtil} = ChromeUtils.import('resource://gre/modules/NetUtil.jsm');
++const dragService = Cc["@mozilla.org/widget/dragservice;1"].getService(
++  Ci.nsIDragService
++);
++const obs = Cc["@mozilla.org/observer-service;1"].getService(
++  Ci.nsIObserverService
```

Comment:
nit: obs -> observerService ?

Reply:
they use obs other places in their code. I firefox style makes it easy to see where this is copy pasted from.